controllers/logoutController.js

```diff
@@ -0,0 +1,14 @@
+const User = require('../models/userModel');
+const session = ('express-session');
+const express = ('express');
+
+const logout = (req, res) => {
+    req.session.regenerate(err => {
```

Comment:
What is regenerate? I thought the method was destroy?

I could be wrong. Let me check on that...

Yup I think you should use destroy in this case